### PR TITLE
VEN-1318 | Refactor Pagination Component

### PR DESCRIPTION
**Pull request by Nizar-Rahme**
## Description :sparkles:
- Get rid of `react-paginate` dependency and implement custom paginator
- Update relevant snapshots and unit tests

### Closes :no_good_woman:
[VEN-1318](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1318): Admin needs to be able to select a specific page

### Automated tests :gear:️
- Updated all relevant tests and snapshots

### Manual testing :construction_worker_man:
- Go to any page that has a table with a paniator below
- The pagination should work as expected 

Note: the number of the page count is clickable and should direct the user to the last page

## Screenshots :camera_flash:
image

### CODE REVIEW

File: src/common/pagination/Pagination.tsx [hunk 1]
```diff
@@ -1,48 +1,69 @@
...
 import { IconArrowLeft, IconArrowRight } from 'hds-react';
 
 import Text from '../text/Text';
 import styles from './pagination.module.scss';
 
-export type PaginationProps = Pick<ReactPaginateProps, 'pageCount' | 'onPageChange' | 'initialPage' | 'forcePage'> & {
+export interface PaginationProps {
   className?: string;
-};
+  forcePage: number;
```
Comment (TatuArvela) [inline src/common/pagination/Pagination.tsx:11]: What does `forcePage` mean? Would `initialPage` be a good term for this?

**Review by TatuArvela — APPROVED** [review]
Great job! 🏆 Left one comment

File: src/common/pagination/Pagination.tsx (hunk 1 above)
Comment (Nizar-Rahme) [reply src/common/pagination/Pagination.tsx:11]: It was a leftover from the `react-paginate` library, I renamed it to `pageIndex`

**Comment by Nizar-Rahme** [comment]
I updated the styles for more consistency across the browsers, if you want to give it another look @TatuArvela

File: src/common/pagination/Pagination.tsx [hunk 2]
```diff
@@ -1,48 +1,71 @@
...
 
+  const options: JSX.Element[] = [];
+
+  for (let pageIndex = 0; pageIndex < pageCount; pageIndex += 1) {
+    options.push(
+      <option key={pageIndex} value={pageIndex}>
+        {pageIndex + 1}
+      </option>
+    );
+  }
```
Comment (TatuArvela) [inline src/common/pagination/Pagination.tsx:27]: This seems to shadow the `pageIndex` prop

**Review by TatuArvela — APPROVED** [review]
Good job! 👍 Added one comment

File: src/common/pagination/Pagination.tsx (hunk 2 above)
Comment (Nizar-Rahme) [reply src/common/pagination/Pagination.tsx:27]: Good catch! Renamed.

**Pull request merged**